README.md

```diff
@@ -121,16 +137,16 @@ not required):
        "irods_default_resource" : "demoResc"
    }
    ```
-   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request.
+   Note that the *"irods_host"* and *"irods_port"* identify the iRODS server that the DSI will contact during each request. Be sure to set the *irods_default_resource*, this variable is not set when you create the file with *iinit* or when you copy it over from another user.
    
 2. As the user who runs the GridFTP server, try an `ils` icommand to verify that 
    the information set in the *irods_environment.json* are fine. If needed, perform 
    an `iinit` to authenticate the iRODS user. 
 
-2. Add the following lines to the GridFTP configuration file ( tipically 
-   *$GLOBUS_LOCATION/etc/gridftp.conf* ):
+3. Add the following lines to the GridFTP configuration file (typically 
+   *$GLOBUS_LOCATION/etc/gridftp.conf*):
    ```
-   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI"
+   $LD_LIBRARY_PATH "$LD_LIBRARY_PATH:/preferred_path/iRODS_DSI/B2STAGE-GridFTP/"
```

Comment:
Ok, problem solved. 
The *so* files are placed in both folders, the *B2STAGE-GridFTP* and the *iRODS_DSI* folder. However you cannot mix them. I.e. when the gridftp.conf uses the *so* files in *iRODS_DSI* the path exported in **LD_PRELOAD** in *globus-gridftp-server* also needs to come from *iRODS_DSI* and not from the *B2STAGE-GridFTP* folder.